API/Data.MSSQL/Migrations/201909261840016_initial.cs

```diff
@@ -1,4 +1,4 @@
-namespace Data.Npgsql.Migrations
+namespace Data.MSSQL.Migrations
```

Comment:
Weird at bruger MSSQL istedet for MySQL. Det er dog auto generet og it worked for us. No need to change :)